### PR TITLE
Use non privileged tcp port in examples

### PR DESCRIPTION
**Pull request by tiagocoutinho**
It would be nice if the examples work out of the box without the user needing access to tcp port 502.

I propose using `15020` in this PR but I could change to something else if you prefer

### CODE REVIEW

**Comment by OrangeTux** [comment]
Thanks for you suggestion!

I don't like to change to default port of the client, as Modbus server are usually listening to port 502.
Changing only the port of the server would make the example server incompatible with the example client. Therefore I also don't want to change the port of the example server.

But I realize it's not very friendly to just let the server crash. Therefore I suggest 2 thinks to make the server more user friendly, but keeping default port at 502.

1) Catch the exception `PermissionError`, print a user friendly message to the user and exit.
2) Add am option to modify the used port from the command line.

**Comment by tiagocoutinho** [comment]
Sounds good!
Should the README examples be changed as well? Or, for the sake of readability, you prefer to keep them as is?

**Comment by OrangeTux** [comment]
Please update the README.md as well. I expect that a significant amount of users just copy the code from the README.md. Although I don't have proof to back it up

**Comment by tiagocoutinho** [comment]
I think it should be ready to review now

File: README.rst [hunk 1]
```diff
@@ -82,26 +97,34 @@ Doing a Modbus request requires even less code:
...
+    port = int(port)
 
     # Returns a message or Application Data Unit (ADU) specific for doing
     # Modbus TCP/IP.
     message = tcp.write_multiple_coils(slave_id=1, starting_address=1, values=[1, 0, 1, 1])
 
-    # Response depends on Modbus function code. This particular returns the
-    # amount of coils written, in this case it is.
-    response = tcp.send_message(message, sock)
+    with create_connection((host, port)) as sock:
```
Comment (OrangeTux) [inline README.rst:123]: TIL that a `socket.Socket` object can be used as a context manager. Thanks!

**Review by OrangeTux — APPROVED** [review]

**Pull request merged**